scripts/linters/pylint_extensions.py

```diff
@@ -385,6 +407,102 @@ def check_functiondef_params(self, node, node_doc):
             node_doc, node.args, node,
             accept_no_param_doc=node_allow_no_param)
 
+    def check_arg_section_indentation(self, node):
+        """Checks whether the function argument definitions (Args: section) are
+        indented properly. Parameters should be indented by 4 relative to the
+        'Arg: ' line and any wrap-around in_descriptions should be indented by
+        8. The rest of the docstring section checks are in DocstringChecker.
+
+        Args:
+            node: astroid.scoped_nodes.Function. Node for a function or
+                method definition in the AST.
+        """
+        arguments_node = node.args
+        expected_argument_names = set(
+            None if (arg.name in self.not_needed_param_in_docstring)
+            else (arg.name + ':') for arg in arguments_node.args)
+        currently_in_args_section = False
+        # When we are in the args section and a line ends in a colon,
+        # we can ignore the indentation styling in the next section of
+        # in_description, hence a freeform section.
+        currently_in_freeform_section = False
+        args_indentation = 0
+        if node.doc:
+            # Process the docstring line by line.
+            for line in node.doc.splitlines():
+                line_with_no_leading_whitespaces = line.lstrip()
+                current_line_indentation = (
+                    len(line) - len(line_with_no_leading_whitespaces))
+                parameter = re.search(
+                    '^[^:]+:',
+                    line_with_no_leading_whitespaces)
+                # Check for empty lines and ignore them.
+                if len(line.strip()) == 0:
+                    continue
+                # Check for the start of an Args: section and check the correct
+                # indentation.
+                if line_with_no_leading_whitespaces.startswith('Args:'):
+                    args_indentation = current_line_indentation
+                    #The current args indentation is incorrect.
```

Comment:
Add space after #. Ditto elsewhere. (In general, follow existing style.)

Reply:
Done